aws-distro-opentelemetry-node-autoinstrumentation/test/patches/aws/services/bedrock.test.ts

```diff
@@ -397,6 +397,60 @@ describe('BedrockRuntime', () => {
       expect(invokeModelSpan.kind).toBe(SpanKind.CLIENT);
     });
 
+    it('Add Amazon Nova model attributes to span', async () => {
+      const modelId: string = 'amazon.nova-pro-v1:0';
+      const prompt: string = 'Campfire story';
+      const mockRequestBody: string = JSON.stringify({
+        inputText: prompt,
+        inferenceConfig: {
+          max_new_tokens: 500,
+          temperature: 0.9,
+          top_p: 0.7,
+        },
+      });
+      const mockResponseBody: any = {
+        output: { message: { content: [{ text: '' }], role: 'assistant' } },
+        stopReason: 'max_tokens',
+        usage: { inputTokens: 432, outputTokens: 681 },
+
+        request: {
+          commandInput: {
+            modelId: modelId,
+          },
+        },
+      };
+
+      nock(`https://bedrock-runtime.${region}.amazonaws.com`)
+        .post(`/model/${encodeURIComponent(modelId)}/invoke`)
+        .reply(200, mockResponseBody);
+
+      await bedrock
+        .invokeModel({
+          modelId: modelId,
+          body: mockRequestBody,
+        })
+        .catch((err: any) => {});
+
+      const testSpans: ReadableSpan[] = getTestSpans();
+      const invokeModelSpans: ReadableSpan[] = testSpans.filter((s: ReadableSpan) => {
+        return s.name === 'BedrockRuntime.InvokeModel';
+      });
+      expect(invokeModelSpans.length).toBe(1);
+      const invokeModelSpan = invokeModelSpans[0];
+      expect(invokeModelSpan.attributes[AWS_ATTRIBUTE_KEYS.AWS_BEDROCK_AGENT_ID]).toBeUndefined();
+      expect(invokeModelSpan.attributes[AWS_ATTRIBUTE_KEYS.AWS_BEDROCK_KNOWLEDGE_BASE_ID]).toBeUndefined();
+      expect(invokeModelSpan.attributes[AWS_ATTRIBUTE_KEYS.AWS_BEDROCK_DATA_SOURCE_ID]).toBeUndefined();
+      expect(invokeModelSpan.attributes[AwsSpanProcessingUtil.GEN_AI_SYSTEM]).toBe('aws_bedrock');
```

Comment:
Small change. Can you update this to assert `aws.bedrock` for this new model?

We changed this recently to align better with upstream Otel semantic conventions.

I raised this [PR](https://github.com/aws-observability/aws-otel-js-instrumentation/pull/134) to make the implementation change and updated the existing tests, so no need to worry about those.

Reply:
Change pushed!